examples/msbr/geometry/msbr_full.ini

```diff
@@ -162,10 +148,9 @@ cell 848  4  moder -48                                       % replay part of le
 cell 849  4  moder -49                                       % replay part of right-bottom graphite element
 cell 850  4  moder -50                                       % replay part of right-upper graphite element
 cell 851  4  moder -51                                       % replay part of left-bottom graphite element
-cell 852  4  outside   43                      		     % Outside world
 
 % ------ Vacuum cell
-cell 57  5  void -3
+cell 57  5  void
 
```

Comment:
That's not quite right. I've removed the surface that bounds the void cell (Universe 5). Universe 5 is only used in lattice elements in the main lattice, so having that universe bounded by a surface that is equivalent to the lattice element boundary is redundant. That's why i removed the bounding surface for Universe 5, since the lattice does the bounding instead.